### DISCUSSION TITLE
How to add tags for graph bar to Y-axis

### DISCUSSION THREAD

**Discussion by Wannapayeot** [frangoteam/FUXA#1287] [Q&A]
I want to add a tag to the Y axis, but I can't find the Y axis settings. What do I do? Please give me some advice.
![fuxas](https://github.com/user-attachments/assets/e004aaa8-5e16-4eb1-8dcf-569c697ab387)

**Comment by unocelli** [comment]
Hi, In the Y axis should be your tags values! what you will configure in the Y axis?

**Reply by Wannapayeot** [reply]
![image](https://github.com/user-attachments/assets/7d49c467-0e82-49d8-ae73-cc47b50d8913)
In the example image, I want to put the date on the Y axis, but I can't find a setting that allows me to put the date on the Y axis.

**Reply by unocelli** [reply]
you mean X axis or I don't understand your graph
